setup-tpm2-totp.sh

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
```suggestion
set -euo pipefail

```

Should this be executable? The other `setup-*.sh` ones aren't, but…